doc/gitian-building.md

```diff
@@ -262,12 +262,12 @@ Then set up LXC and the rest with the following, which is a complex jumble of se
 # the version of lxc-start in Debian 7.4 needs to run as root, so make sure
```

Comment:
should we address the comment about debian 7.4 also, given that we've moved to 8.2